app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name = command.split()[1]
+    new_path = command.split()[2]
+    path_list = new_path.split("/")
```

Comment:
Instead of splitting the path manually, use `os.path.join` to concatenate parts of the path. This ensures cross-platform compatibility, as per the checklist item: 'Use os.path.join to concatenate parts of the path.'